### PR TITLE
feat: display "unsupported file type" error, show time to complete

### PR DESCRIPTION
**Pull request by okdargy**
image
image

General improvements to make the bot better!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,10 +3,9 @@
   "version": "2.1.0",
   "description": "your go-to bot for converting audio into video, tailored specifically for the convenience of discord mobile users.",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
     "build": "tsc",
     "start": "node dist/index.js",
-    "setup": "pnpx mkdirp assets/temp/in && npx mkdirp assets/temp/out",
+    "setup": "npx mkdirp assets/temp/in && npx mkdirp assets/temp/out",
```
Comment (veryCrunchy) [inline package.json]: Ples no

File: src/pages/help.ts [hunk 2]
```diff
@@ -96,7 +96,7 @@ export function getCategoryPage(
     );
 
   // Get current offset
-  let offset = parseInt(currentOffset);
+  let offset = parseInt(currentOffset ?? "0");
```
Comment (veryCrunchy) [inline src/pages/help.ts]: This case is already handled by the NaN check.
Why we do a NaN check.. I can't remember and I'm too unbothered to find out, HA
```suggestion
  let offset = parseInt(currentOffset);
```

Comment (okdargy) [reply src/pages/help.ts]: When building, TypeScript complains sooo... it was just to satisfy the build

Comment (veryCrunchy) [reply src/pages/help.ts]: Anything to satisfy the build 😩

Comment (veryCrunchy) [reply src/pages/help.ts]: Fine for now though since I already committed upsi, rewrite bouta come in

**Review by veryCrunchy — APPROVED** [review]
perfection (through dirty glasses)👌

**Comment by indexxing** [comment]
skibidi

**Pull request merged**